lib/pacemaker/pcmk_output.c

```diff
@@ -1161,7 +1175,8 @@ rsc_action_default(pcmk__output_t *out, va_list args)
             STOP_SANITY_ASSERT(__LINE__);
 
         } else if (moving && current) {
-            rc = out->message(out, "rsc-action-item", pcmk_is_set(rsc->flags, pe_rsc_failed)? "Recover" : "Move",
+            rc = out->message(out, "rsc-action-item",
+                              pcmk_is_set(rsc->flags, pe_rsc_failed)? "Recover" : "Move",
```

Comment:
Did this one fit, or just leaving it for now? Could do a local bool variable or similar

Reply:
I left a bunch like this that would require putting things in variables, but just out of laziness :) I'll take care of them